proto/frequenz/api/common/v1/pagination.proto

```diff
@@ -0,0 +1,34 @@
+// Parameters for pagination.
+//
+// Copyright 2023 Frequenz Energy-as-a-Service GmbH
+//
+// Licensed under the MIT License (the "License");
+// you may not use this file except in compliance with the License.
+
+syntax = "proto3";
+
+package frequenz.api.common.v1.pagination;
+
+// A message defining parameters for paginating list requests.
+message PaginationParams {
+  // [Optional]
+  // The maximum number of results to be returned per request.
+  // Defaults to 50 if omitted.
+  uint32 page_size = 1;
+
+  // [Optional]
+  // The token identifying a specific page of the list results.
+  string page_token = 2;
+}
+
+// A message providing metadata about paginated list results.
+message PaginationInfo {
+  // [Optional]
+  // The total number of items that match the filter criteria defined in the
+  // request.
+  uint32 total_filtered_items = 1;
```

Comment:
Is there a reason to not call this `total_items`?

Reply:
i thought it'd be more clear to add the filtered. But it's probably clear from the context. So rather `total_items` ?

Reply:
yeah I would keep it lean.